README.md

```diff
@@ -34,16 +34,32 @@ $ poetry run python -m pytest tests/e2e
 ```
 ## Running the app
 
-Once the all dependencies have been installed and all tests succeed, start the Flask app in development mode within the poetry environment by running:
+Once the all dependencies have been installed and all tests succeed, there are multiple ways to start the app.
+
+Firstly, the app can be started in development-mode inside a docker container by running:
+```bash
+$ docker-compose up
+```
+
+\
+Secondly, you can run the app in production-mode by running:
 ```bash
-$ poetry run flask run
+$ docker build --target production --no-cache --tag todo-app:prod .
 ```
-or you can run the app inside a VM by running:
+to build a production-mode docker image from ```Dockerfile```, followed by
+```bash
+$ docker run --env-file .env -p 5000:5000 --mount type=bind,source="$(pwd)",target=/DevOps-Course-Starter todo-app:prod
```

Comment:
Might be worth stressing which shells this command will work with (this is another reason why compose files are better in general)